client-ui/src/app/generated/models.ts

```diff
@@ -0,0 +1,14 @@
+export { Attribute } from './models/attribute';
+export { DataSourceInfo } from './models/data-source-info';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4584725)